tor/strings/en_US.yml

```diff
@@ -99,6 +99,8 @@ responses:
     claimed_other_user: |
       Sorry, this post is claimed by someone else, so you cannot unclaim it!
   general:
+    blacklisted: |
```

Comment:
Historically we've built it so that they are just ignored by the bots... I think that unless we have a good reason to inform people of this fact, we should just continue ignoring them.

Reply:
You're right, we could also do this. That being said, wouldn't it be easier to just make the respective user aware of the fact?

Reply:
Blacklisting should be reserved exclusively for people who are abusing the platform, and it's not an automatic process -- it's one that we trigger manually. I perceive that we should follow the reddit shadowban system -- let them scream into the void in the hopes that something answers until they go away. We will have already made our intentions clear long before blacklisting becomes an option.